docs/examples/general/conditionals.ipynb

```diff
@@ -153,7 +153,7 @@
     "\n",
     "### Requirements\n",
     "There are several requirements that needs to be met when using the conditionals within DALI pipeline:\n",
-    "1. The `if` condition must be a DALI DataNode and the underlying data must be of type Bool.\n",
+    "1. The `if` condition must be a DALI DataNode and the underlying data must be of type `BOOL` or other numerical type that can be evaluated to True or False in accordance with Python semantics.\n",
```

Comment:
I think that at this point there's no reason to mention bool.
```suggestion
    "1. The `if` condition must be a DALI DataNode and the underlying data must be of a numerical type that can be evaluated to True or False in accordance with Python semantics.\n",
```
This is a nitpick / thing to consider, nothing serious.

Reply:
CI is super lazy, I will update it when I revisit the docs about the logical expressions `and`, `or` and `not`.